scrapegraphai/nodes/text_node.py

```diff
@@ -0,0 +1,61 @@
+""" 
+Module for TextNode
+"""
+from .base_node import BaseNode
+
+
+class TextNode(BaseNode):
+    """
+   A node for loading the text in the state
+
+    This node acts as a starting point in many scraping workflows, preparing the state
+    with the necessary HTML content for further processing by subsequent nodes in the graph.
+
+    Attributes:
+        node_name (str): The unique identifier name for the node.
+        node_type (str): The type of the node, defaulting to "node". This categorization
+                         helps in determining the node's role and behavior within the graph.
+                         The "node" type is used for standard operational nodes.
+
+    Args:
+        node_name (str): The unique identifier name for the node. This name is used to
+                         reference the node within the graph.
+        node_type (str, optional): The type of the node, limited to "node" or
+                                   "conditional_node". Defaults to "node".
```

Comment:
I would rewrite the documentation for this node in order to make as cleas as possible that takes general text, the parameter url for this use case could cause misunderstanding, i would change to file